src/UI/Actions.hs

```diff
@@ -498,7 +493,7 @@ class ViewTransition (v :: ViewName) (v' :: ViewName) where
 instance ViewTransition v v where
 
 instance ViewTransition 'Mails 'Threads where
-  transitionHook _ _ = set (asViews . vsViews . at Mails . _Just) listOfMailsView
+  transitionHook _ _ = set (asViews . vsViews . ix Mails) listOfMailsView
```

Comment:
Wow, it is nice that HLint groks some lens stuff :)

Reply:
Yeah hey. I like it.